returnn/tf/network.py

```diff
@@ -15,7 +15,7 @@
 from returnn.tf.layers.basic import LayerBase, get_layer_class
 import returnn.tf.compat as tf_compat
 import returnn.tf.util.basic as tf_util
-from returnn.tf.util.basic import Data, DimensionTag, reuse_name_scope, VariableAssigner
+from returnn.tf.util.basic import Data, DimensionTag, reuse_name_scope, VariableAssigner, get_checkpoint_filepattern
```

Comment:
I prefer to not add too much global imports except for the very common things like `Data`, esp not for rarely used functions.
Rather, use `tf_util.`.
(Maybe the other imports here should be cleaned up as well, like `VariableAssigner`. But this is off-topic here.)